Loop/Managers/DeviceDataManager.swift

```diff
@@ -1376,81 +1376,56 @@ extension DeviceDataManager {
             log.error("Remote Notification: Remote Commands not enabled.")
             return
         }
-
```

Comment:
I consider the validation of remote commands to be owned by the domain of the provider (plugin) that produced the command (ex: OTP and Expiration). This allows the providers to have their own validation criteria and Loop to not be constrained by older provider versions.

There is an argument that certain things happen in the provider (maybe OTP) and certain things in Loop (i.e. Expiration and later dose hashing).